{"uuid": "ef0779f4-15b2-45a2-bbcb-c5497564336d", "children": ["533c2781-d753-4c52-bc6e-1a252a44c9d9"], "befores": [{"name": "email_password", "status": "broken", "statusDetails": {"message": "UnboundLocalError: cannot access local variable 'access_token' where it is not associated with a value\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/dchernukha/PycharmProjects/Diplom/Diplom_3/conftest.py\", line 34, in email_password\n    email_password, access_token = create_new_user_and_return_user_data()\n                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/dchernukha/PycharmProjects/Diplom/Diplom_3/helpers.py\", line 54, in create_new_user_and_return_user_data\n    return email_password, access_token\n                           ^^^^^^^^^^^^\n"}, "steps": [{"name": "Регистрация пользователя и сохранение его данных.", "status": "broken", "statusDetails": {"message": "UnboundLocalError: cannot access local variable 'access_token' where it is not associated with a value\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/dchernukha/PycharmProjects/Diplom/Diplom_3/helpers.py\", line 54, in create_new_user_and_return_user_data\n    return email_password, access_token\n                           ^^^^^^^^^^^^\n"}, "steps": [{"name": "Генерация тела запроса со всеми параметрами для создания пользователя.", "status": "passed", "start": 1732720785834, "stop": 1732720785840}, {"name": "Создание пользователя через POST-запрос.", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'lisa86@example.com', 'password': '$^PFRAvr9Q', 'name': 'masseymichael'}"}], "start": 1732720785840, "stop": 1732720786183}], "start": 1732720785834, "stop": 1732720786183}], "start": 1732720785834, "stop": 1732720786183}], "start": 1732720785834, "stop": 1732720786192}